src/instrumentsscene/qml/MuseScore/InstrumentsScene/internal/InstrumentsControlPanel.qml

```diff
@@ -32,6 +32,7 @@ RowLayout {
     property bool isMovingDownAvailable: false
     property bool isRemovingAvailable: false
     property bool isAddingAvailable: value
+    property bool isInstrumentSelected: value
```

Comment:
I just notice that this code contains a somewhat funny mistake: instead of `value`, it should be `false`. (It doesn't cause problems because `value` is `undefined` which converts to `false`, and because the property is overridden anyway, but still it would be good to fix it.)

Why someone accidentally wrote `value`? I think that's because that's what you get when using Qt Creator's auto-complete snippet for properties!

Reply:
Yeah, you're right.) Just fixed it